### ISSUE TITLE
[FEATURE] IP masking

### ISSUE THREAD

**Issue by GigiaJ** [OSRSB/OsrsBot#7]
Currently each individual client is not masked at all, so the end result would look like us running fifty clients on one computer. If possible we need to develop a method to mask IP addresses. In the past I've managed to add run parameters to the shortcuts and programs to manipulate them into connecting to a proxy server. I'll look into incorporating a past solution similar to this or coding up a method to allow the connection between each client and Jagex' servers to be route-able through a proxy service.

**Comment by f0rmatme** [comment]
https://github.com/open-osrs/runelite/pull/447/files

**Comment by GigiaJ** [comment]
Oh wow. Thanks.

I'd like to note this is _part_ of what I was hoping to add. 
I wanted to look into scanning proxy sites and testing each proxy on them to a certain number of pages to find the best one and then applying them to the bot. Could even do this prior to bot initialization as a preload and just hold the best six (or whatever is decided for the maximum number of bots.) 
TL;DR 
A proxy for each bot instance

I can definitely go ahead and implement the above. I will be keeping this open until I either determine my goal above is not easily possible or it is implemented.

Thanks a bunch though!
